tests/js/spec/helpers/formatters.spec.jsx

```diff
@@ -19,5 +19,34 @@ describe('formatters', function() {
         })
       ).toEqual('user (foo@bar.com)');
     });
+
+    it('should show unknown author with email, if email is only provided', function() {
+      expect(
+        userDisplayName({
+          email: 'foo@bar.com',
+        })
+      ).toEqual('Unknown author (foo@bar.com)');
+    });
+
+    it('should show unknown author, if author or email is just whitespace', function() {
```

Comment:
I don't have a strong opinion on this. If an author name is intentionally whitespace, I can adjust the `userDisplayName()` to be as such.

Reply:
I don't think explicit whitespace is a use case we need to afford.